src/network/wait-online/wait-online.c

```diff
@@ -41,6 +42,7 @@ static int help(void) {
                "     --ignore=INTERFACE     Don't take these interfaces into account\n"
                "  -o --operational-state=OPERSTATE\n"
                "                            Required operational state\n"
+               "     --any                  Wait for until one of the interfaces to be online\n"
```

Comment:
Wait for until → Wait until at least
to be online → is online

i.e. `Wait until at leaset one of the interfaces is online`